daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/SpeedyTest.scala

```diff
@@ -316,6 +318,14 @@ class SpeedyTest extends WordSpec with Matchers {
 
     }
 
+    /*
+    Some(SELabelClosure(LfDefRef(-pkgId-:M:p_1_0),
+        SELet1General(SEVal(LfDefRef(-pkgId-:M:origin)),SEAppAtomicSaturatedBuiltin(SBRecUpd(-pkgId-:M:Point,0),[SELocS(1),SEValue(SInt64(1))])))) did not equal
+    Some(SELet1General(SEVal(LfDefRef(-pkgId-:M:origin)),SEAppAtomicSaturatedBuiltin(SBRecUpd(-pkgId-:M:Point,0),[SELocS(1),SEValue(SInt64(1))])))
```

Comment:
Why is this test commented out?

Reply:
They have nothing to do here. 
I remove them in #7450

Reply:
I see. Thanks for the explanation.